packages/@sanity/portable-text-editor/test/setup/collaborative.jest.env.ts

```diff
@@ -24,29 +18,71 @@ const WEB_SERVER_ROOT_URL = 'http://localhost:3000'
 // eslint-disable-next-line no-process-env
 const DEBUG = process.env.DEBUG || false
 
-// Wait this long for selections and a new doc revision to appear in the clients.
-const SELECTION_TIMEOUT_MS = 500
+// Wait this long for selections to appear in the browser
+// This should be set high to support slower host systems.
+const SELECTION_TIMEOUT_MS = 2000
 
 // How long to wait for a new revision to come back to the client(s) when patched through the server.
-// Wait for patch debounce time and some slack for selection adjustment time for everything to be ready
-const REVISION_TIMEOUT_MS = 1000
+// This should be set high to support slower host systems.
+const REVISION_TIMEOUT_MS = 5000
 
 export default class CollaborationEnvironment extends NodeEnvironment {
   private _browserA?: Browser
   private _browserB?: Browser
   private _pageA?: Page
   private _pageB?: Page
+  private _contextA?: BrowserContext
+  private _contextB?: BrowserContext
+
+  // public async setup(): Promise<void> {
```

Comment:
can these comments be removed? or add a comment about why they need to stay?

Reply:
I want to keep them there for future reference. Added a comment.